f --git a/templates/flag.html.twig b/templates/flag.html.twig
index 494dc90..3f76b70 100644
--- a/templates/flag.html.twig
+++ b/templates/flag.html.twig
@@ -24,8 +24,8 @@
 {# Set the remaining Flag CSS classes. #}
 {% set classes = [
 'flag',
-'flag-' ~ flag.id,
-'flag-' ~ flag.id ~ '-' ~ flaggable.id,
+'flag-' ~ flag.id(),
+'flag-' ~ flag.id() ~ '-' ~ flaggable.id(),
 action_class
 ] %}
 {% set attributes = attributes.addClass(classes) %}
